spec/rubocop/cop/style/commented_keyword_spec.rb

```diff
@@ -246,6 +246,9 @@ class Y
         def y #: String
         end
       end
+
+      class Y < X #[String]
+      end
```

Comment:
Could you make it a separate test case?

Reply:
@koic 

I wrote separate test cases for subclass definition.